### PR TITLE
Bump odo version to 1.0.0-beta6

### PR DESCRIPTION
**Pull request by dharmit**
## Enhancements

* Support for `odo debug`. `odo` now lets users debug the application running in the pod on OpenShift (#2043)
* Add json output for `odo catalog list services`. This helps the consumers of `odo`, i.e., Adapters (#2066)
* Add json output for `odo service list`. This helps the consumers of `odo`, i.e., Adapters (#2098)
* Add RPM spec for odo. Very soon, an RPM for `odo` will be available and `odo` could be installed with `yum/dnf install odo`  (#2107)
* Replace dumb-init with go-init. To avoid having to maintain our own RPM for dumb-init  (#2130)
* Refactor catalog code  (#2133)
* Print environment variables when linking a component with another component/service (#2035)

## Bug Fixes
* Fix populating parameters of service when created in non-interactive mode. `odo service create` didn't pick-up the values provided for parameters with `-p` flag (#2040)
* Fix `odo list` for component that's not pushed. `odo list` output was empty for the components that have not been pushed (#2046)
* Removes overlapping messages for odo git components (#2091)

## Testing
* Fixing e2e vs. integration naming (#1878)
* Run tests on OpenShift 4.2 (#2059)
* Add "pretty" library for testing (#2122)
  This makes debugging less painful to the eyes.

## Documentation
*  Add development doc on machine readable output (#2123)

### CODE REVIEW

**Comment by dharmit** [comment]
Need to merge #2177 first.

**Comment by kadel** [comment]
`✗  Failed To Update Config To Component Deployed`
#1981 
/retest

**Comment by amitkrout** [comment]
/approve
ping @girishramnani

**Comment by dharmit** [comment]
Should I not be adding Enhancements, Fixes, etc. so that they show up on release page? Give me some time and I'll update the description of the PR with those details. 

/hold

**Comment by dharmit** [comment]
/hold cancel

ping @girishramnani @kadel

**Comment by girishramnani** [comment]
A bit more description per point?

**Comment by dharmit** [comment]
> A bit more description per point?

Are you asking me to write an essay in Changelog? :roll_eyes:

**Comment by cdrage** [comment]
I think that writing it in vim may have messed up the spacing, but when you do the release, turn:

```sh
Add json output for odo catalog list services (#2066)
This helps the consumers of odo, i.e., Adapters
```

To:

```sh
Add json output for odo catalog list services. This helps the consumers of odo, i.e., Adapters (#2066)
```

Similar to the previous format, have it so that it's 1 sentence and that the link to the PR is at the end.

**Comment by dharmit** [comment]
> I think that writing it in vim may have messed up the spacing, but when you do the release, turn:
> 
> ```shell
> Add json output for odo catalog list services (#2066)
> This helps the consumers of odo, i.e., Adapters
> ```
> 
> To:
> 
> ```shell
> Add json output for odo catalog list services. This helps the consumers of odo, i.e., Adapters (#2066)
> ```
> 
> Similar to the previous format, have it so that it's 1 sentence and that the link to the PR is at the end.

No, it wasn't vim. I edited the description on GH. But yeah I felt like it was a bit out of place. I'll modify that.

File: docs/cli-reference.adoc [hunk 1]
```diff
@@ -398,6 +406,28 @@ By default, builder images will be used from the current namespace. You can expl
 
 A full list of component types that can be deployed is available using: 'odo catalog list'
 
+[[debug]]
+debug
```
Comment (kadel) [inline docs/cli-reference.adoc]: Why is debug section empty?

Comment (dharmit) [reply docs/cli-reference.adoc]: TBH, I'm not sure if that's something I should have added or `make generate-cli-reference` should have.

Comment (kadel) [reply docs/cli-reference.adoc]: OK, no examples in the command-line. It will have to be fixed there.

/lgtm

**Comment by kadel** [comment]
/retest

```
error: error pulling image registry.svc.ci.openshift.org/ci/clonerefs@sha256:99ef47c07da4a436f069583cb2326e9e8c2b0f296400eee48e865872a17a33eb: Get https://registry.svc.ci.openshift.org/v2/ci/clonerefs/manifests/sha256:99ef47c07da4a436f069583cb2326e9e8c2b0f296400eee48e865872a17a33eb: net/http: TLS handshake timeout
```

**Pull request merged**